firefox/manifest.json

```diff
@@ -1,29 +0,0 @@
-{
-  "manifest_version": 2,
-  "name": "MakeGithubGreatAgain",
-  "version": "1.0.0",
-  "description": "Extension to make GitHub great again!.",
-  "icons": {
-    "16": "icon_16.png",
-    "48": "icon_48.png",
-    "128": "icon_128.png"
-  },
-  "browser_action": {
-    "default_icon": "icon_16.png"
-  },
-  "content_scripts": [
-    {
-      "matches": [
-        "*://github.com/*",
-        "*://*.github.com/*"
-      ],
-      "js": [
-        "firefox.js"
-      ],
-      "css": [
-
```

Comment:
No CSS was being apllied here, so the latest official firefox extension isn't working.